core/Cargo.toml

```diff
@@ -12,6 +12,10 @@ description = "A modern JavaScript/TypeScript runtime built with V8, Rust, and T
 [lib]
 path = "lib.rs"
 
+[features]
+default = ["v8_use_custom_libcxx"]
```

Comment:
Why do you want to make it a default feature?

Reply:
It is the current default. I think it could be contentious to change it.

Reply:
Is it? Adding `default-features = false` to `v8` imports, suggests that it's not, or maybe I'm misunderstanding something?

Reply:
It's a default feature of v8 https://github.com/denoland/rusty_v8/pull/924, so as of today it's the default of deno core.

With that said, the only way to make it optional to users of deno core is to make it `default-features=false` at the dependency level but enable via a feature. This new feature can then be enabled by default (preserving the status quo) but can be disabled by deno core users.